spanner/tests/system.py

```diff
@@ -75,9 +85,6 @@ def setUpModule():
     Config.INSTANCE_CONFIG = configs[0]
     config_name = configs[0].name
 
-    def _list_instances():
-        return list(Config.CLIENT.list_instances())
-
     instances = retry(_list_instances)()
```

Comment:
I don't see how a one-liner defined immediately above its only point-of-use is harder to reason about than the same function moved further away, and made (pointlessly, otherwise) a method.

Go ahead and leave it as is.  However, if we are adopting that as a standard (e.g., you ask for me to change it on my PRs every time you see it), then we should be linting for it:  otherwise it is a pure *de gustibus* quibble.